usaspending_api/common/mixins.py

```diff
@@ -254,6 +256,70 @@ class SuperLoggingMixin(LoggingMixin):
 
     events_logger = logging.getLogger("events")
 
+    """Mixin to log requests - customized to disable DB logging, remove this method to re-enable"""
+    def initial(self, request, *args, **kwargs):
+        """Set current time on request"""
+
+        # check if request method is being logged
+        if self.logging_methods != '__all__' and request.method not in self.logging_methods:
+            super(LoggingMixin, self).initial(request, *args, **kwargs)
+            return None
+
+        # get IP
```

Comment:
This data is only available via events.log, and only logs the IP of the address sending the request. I'm not sure what privacy concern there would be, as when someone makes a request to a website you're always going to be giving them your IP.